app/models/event.rb

```diff
@@ -99,7 +99,8 @@ class Event < ApplicationRecord
   has_ontology_terms(:scientific_topics, branch: OBO_EDAM.topics)
   has_ontology_terms(:operations, branch: OBO_EDAM.operations)
 
-  validates :title, :url, presence: true
+  validates :title, :url, :start, :end, presence: true
```

Comment:
we shouldn't do this, it might break some scrapers. Let's keep only the URL and title validations.